docker/Dockerfile-debian

```diff
@@ -18,7 +18,10 @@ RUN apt-get update \
     && apt-get install -y python3-dev python3-pip \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip install poetry
+RUN python3 -m pip install --user pipx
+RUN python3 -m pipx ensurepath
+# Make sure this is the same version as in the Alpine-based images
```

Comment:
I don't think this needs to be true. Once `python3-poetry` is available in Debian, we should just use that.